Automattic/wp-calypso
# Subscription Management: Create CommentSettings and UnfollowCommentsButton components

Closes https://github.com/Automattic/wp-calypso/issues/75481
Closes https://github.com/Automattic/wp-calypso/issues/75482

## Proposed Changes

* Creates `CommentSettings` and `UnfollowCommentsButton` components
* Doesn't integrate with mutations
* Follows the same approach as `SiteSettings`, using `onUnfollow` handler to perform the unfollow action and `unfollowing` prop for loading state

## Testing Instructions

- Run this PR on your local env
- For testing purposes, apply this component to `client/landing/subscriptions/site-list/site-row.tsx`
```
<CommentSettings onUnfollow={ () => undefined } unfollowing={ false } />
```
- Verify if the component is rendered as expected

<img width="328" alt="image" src="https://user-images.githubusercontent.com/3113712/230998406-731b5ac2-7adc-4c13-8a95-1fb90ba8caa1.png">


## Pre-merge Checklist
